src/button-dropdown/styles.scss

```diff
@@ -10,6 +10,11 @@ $dropdown-trigger-icon-offset: 2px;
 
 .button-dropdown {
   display: inline-block;
+
+  &.full-width {
+    inline-size: 100%;
+    text-align: center;
```

Comment:
`text-align` won't do much here, cause it's getting reset in Dropdown.

Reply:
you mean for a use-case without split button? In that case internal button will set all required styles based on `<InternalButton fullWidth={fullWidth}`

Reply:
you're right, removed.